provision/ubuntu/install.sh

```diff
@@ -123,6 +123,14 @@ go version
 sudo sh -c "curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
 sudo chmod +x /usr/local/bin/docker-compose
 
+#Install bpftool
+cd /tmp/
+git clone --depth 1 -b master git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
```

Comment:
Maybe we could use the bpf-next tree for ubuntu-next? Then we could get rid of fetching kernel sources at https://github.com/cilium/packer-ci-build/blob/master/provision/ubuntu/kernel-next.sh#L11.

Reply:
Yes, true, I can try to fetch sources once and just install the kernel in kernel-next image.